### PR TITLE
Portfolio milestone 01

### PR DESCRIPTION
**Pull request by shahadat3669**
### Portfolio: Milestone 01

- [x] GitHub workflow added
- [x] Linter errors fixed
- [x]  index.html added
- [x]  style.css added
- [x] README.md file added
- [x] Completed Html, and CSS for the toolbar and headline section
- [x] README.md file updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  </head>
+
+  <body>
+    <header id="header">
+      <div class="wrapper">
+        <div class="header-inner">
+          <a href="/" class="header-logo">Welcome</a>
+          <a href="#" class="header-hamburger">
+            <img src="dist/images/hamburger.svg" alt="hamburger icon" />
+          </a>
```
Comment (Nemwel-Boniface) [inline index.html]: - [x] For semantic reasons, it is always advisable to wrap your hamburger menu with the nav tag. Kindly wrap it with a nav.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          </ul>
+          <a href="#recent-work" class="headline-scroll">
+            <img
+              src="./dist/images/icon-down.svg"
+              alt="headline scroll icon"
+              class="headline-scroll-icon" />
+          </a>
+        </div>
+      </div>
+    </section>
```
Comment (Nemwel-Boniface) [inline index.html:101]: - [x] Good job using the correct semantic tag for your headline section. I have realized that there is a background image that is present in the original design that is currently missing in your design. I would urge you to include it.

**This is how I am seeing this section:**

image

**This is the desired design with the background images:**

image

**Note:** the difference in the designs is only with the background images. Kindly include them.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @shahadat3669     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your project so far! :clap: You have an amazing design keep it up :fire: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shahadat3669** [comment]
@Nemwel-Boniface updated the files as your suggested.

- [x] Placed the hamburger icon inside the nav element. 
- [x] Tried my best to make the headline background as it is in the Figma file
_I think it is impossible for the background to adjust fully as in the design. because everyone has different screen sizes. For example, on your screenshot, I can see the "Supporting Text" is 7 lines but the design has 6 lines. It increases the div height. What do you say about my thought, let me know. Thanks._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @shahadat3669, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shahadat3669** [comment]
Completed milestone 01

**Pull request merged**